src/docs/firmware/manually-flashing.md

```diff
@@ -6,11 +6,12 @@ Flashing your own firmware has the potential to brick your device. Do not do thi
 
 1. Have your firmware rom. We will assume you already have the file you want to flash for this guide.
 2. Download flashrom, then give it execute permission.
-   - `cd; curl -LO https://tree123.org/chrultrabook/utils/flashrom-libpci38; chmod +x flashrom-libpci38`
+   - For most distributions: `cd; curl -LO https://files.tree123.org/utils/x86_64/gnu/flashrom; chmod +x flashrom`
+   - For musl (Alpine Linux): `cd; curl -LO https://files.tree123.org/utils/x86_64/musl/flashrom; chmod +x flashrom`
 3. Flash your custom ROM
-   - Backup your current rom, just in case things go wrong: `./flashrom-libpci38 -p internal -r current.rom`
+   - Backup your current rom, just in case things go wrong: `./flashrom -p internal -r current.rom`
    - Note: Intel Chromebooks should be flashed with `--ifd -i bios` ex: `sudo ./flashrom -p internal --ifd -i bios -w coreboot.rom`
-   - Flash your custom firmware: `sudo ./flashrom-libpci38 -p internal -w firmware.rom`
+   - Flash your custom firmware: `sudo ./flashrom -p internal -w firmware.rom`
```

Comment:
perhaps out of scope for this change, but the flashing command line should really be an either or: for Intel use this, for AMD/ARM use that. having it in a note when it's the default most people will use seems odd